core/src/components/modal/animations/ios.enter.ts

```diff
@@ -52,14 +52,23 @@ export const iosEnterAnimation = (
       const toPresentingScale = SwipeToCloseDefaults.MIN_PRESENTING_SCALE;
       const finalTransform = `translateY(${modalTransform}) scale(${toPresentingScale})`;
 
+      let initialBorderRadius = '0px';
+      const isIPhone = /iPhone/.test(navigator.userAgent);
+      const safeArea = getComputedStyle(document.documentElement).getPropertyValue('--ion-safe-area-bottom');
```

Comment:
I would like to stay away from using`getComputedStyle` in this context. It will likely cause jank on lower end iPhones. Is there any way we can do this without forcing a reflow like this?

Reply:
You are right, that was not a very good approach. I pushed a new version where it is now calculation the screen ratio. It seems every iPhone today with an edge to edge display has a screen ratio about 0.46 (px). I put this value in an array so that if any new iPhone released with a different ratio, we can simply add it.